pkg/apiserver/logsearch/task.go

```diff
@@ -260,7 +261,7 @@ func (t *Task) searchLog(client diagnosticspb.DiagnosticsClient, targetType diag
 		}
 		for _, msg := range res.Messages {
 			line := logMessageToString(msg)
-			_, err := bufWriter.Write(*(*[]byte)(unsafe.Pointer(&line)))
+			_, err := bufWriter.Write(*(*[]byte)(unsafe.Pointer(&line))) // #nosec
```

Comment:
ditto, this error is checked and why do we need `nosec`?

Reply:
Every `unsafe` api invoke should use `nosec` tag... I'm wondering whether we need to turn on all gosec rules. Obviously, we know what we are doing when we use a specific api.